src/frontier/frontier_cmd_arcade.c

```diff
@@ -0,0 +1,526 @@
+#include "global.h"
+#include "battle/battle_setup.h"
+#include "frontier/frontier_cmd_arcade.h"
+#include "frontier/overlay_80_022340E8.h"
+#include "frontier/overlay_80_02238034.h"
+#include "frontier/overlay_80_02229EE0.h"
+#include "frontier/overlay_80_02239960.h"
+#include "frontier/overlay_80_0222AB40.h"
+#include "frontier/frontier.h"
+#include "launch_application_data.h"
+#include "unk_02096910.h"
+#include "unk_02034354.h"
+#include "unk_02014DA0.h"
+#include "overlay_42.h"
+#include "library/spl_emitter.h"
+
+static void GameBoardArgs_Set(GAME_BOARD_ARGS *args, ArcadeContext *data);
+static void ov80_02233A1C(void *data);
+static void ov80_02233F1C(FrontierContext *ctx, ArcadeContext *arcadeCtx, u32 a2);
+static void ov80_02233F40(struct SPLEmitter *emitter);
```

Comment:
why struct? is this not typedefed?

Reply:
it's a library struct

Reply:
I checked, and it is typedef'd. Use the typedef'd name.